src/renderer/services/wallet/balances.ts

```diff
@@ -176,7 +177,7 @@ export const balancesState$: Observable<BalancesState> = Rx.combineLatest(
   filterEnabledChains({
     THOR: [THOR.balances$],
     BTC: [BTC.balances$, btcLedgerBalance$],
-    ETH: [ETH.balances$],
+    ETH: [ETH.balances$(ERC20Assets)],
```

Comment:
Similar to previous comment:

Use `ETHAssets` + add a note this list is for testnet only.